v2/client/dial.go

```diff
@@ -0,0 +1,161 @@
+package client
```

Comment:
Please can we document why need both connect V1 & V2 ?

Reply:
see above.